src/SamplesApp/UITests.Shared/Windows_Devices/Midi/MidiDeviceEnumeration.xaml.cs

```diff
@@ -0,0 +1,440 @@
+				this.inputDeviceProperties.IsEnabled = false;
+				NotifyUser("Device not found!");
+				return;
+			}
+
+			// Display the found properties
+			DisplayDeviceProperties(devInfo, this.inputDeviceProperties);
+			NotifyUser("Device information found!");
+		}
+
+		/// <summary>
+		/// Change the active output MIDI device
+		/// </summary>
+		/// <param name="sender">Element that fired the event</param>
+		/// <param name="e">Event arguments</param>
+		private void outputDevices_SelectionChanged(object sender, SelectionChangedEventArgs e)
+		{
+			// Get the selected output MIDI device
+			int selectedOutputDeviceIndex = this.outputDevices.SelectedIndex;
+
+			// Try to display the appropriate device properties
+			if (selectedOutputDeviceIndex < 0)
+			{
+				// Clear output device properties
+				this.outputDeviceProperties.Items.Clear();
+				this.outputDeviceProperties.Items.Add("Select a MIDI output device to view its properties");
+				this.outputDeviceProperties.IsEnabled = false;
+				NotifyUser("Select a MIDI output device to view its properties");
+				return;
+			}
+
+			DeviceInformationCollection devInfoCollection = this.midiOutDeviceWatcher.GetDeviceInformationCollection();
+			if (devInfoCollection == null)
+			{
+				this.outputDeviceProperties.Items.Clear();
+				this.outputDeviceProperties.Items.Add("Device not found!");
+				this.outputDeviceProperties.IsEnabled = false;
+				NotifyUser("Device not found!");
+				return;
+			}
+
+			DeviceInformation devInfo = devInfoCollection[selectedOutputDeviceIndex];
+			if (devInfo == null)
+			{
+				this.outputDeviceProperties.Items.Clear();
+				this.outputDeviceProperties.Items.Add("Device not found!");
+				this.outputDeviceProperties.IsEnabled = false;
+				NotifyUser("Device not found!");
+				return;
+			}
+
+			// Display the found properties
+			DisplayDeviceProperties(devInfo, this.outputDeviceProperties);
+			NotifyUser("Device information found!");
+		}
+
+		/// <summary>
+		/// Display the properties of the MIDI device to the user
+		/// </summary>
+		/// <param name="devInfo"></param>
+		/// <param name="propertiesList"></param>
+		private void DisplayDeviceProperties(DeviceInformation devInfo, ListView propertiesList)
+		{
+			propertiesList.Items.Clear();
+			propertiesList.Items.Add("Id: " + devInfo.Id);
+			propertiesList.Items.Add("Name: " + devInfo.Name);
+			propertiesList.Items.Add("IsDefault: " + devInfo.IsDefault);
+			propertiesList.Items.Add("IsEnabled: " + devInfo.IsEnabled);
+			propertiesList.Items.Add("EnclosureLocation: " + devInfo.EnclosureLocation);
+
+			// Add device interface information
+			propertiesList.Items.Add("----Device Interface----");
+			foreach (var deviceProperty in devInfo.Properties)
+			{
+				propertiesList.Items.Add(deviceProperty.Key + ": " + deviceProperty.Value);
+			}
+
+			propertiesList.IsEnabled = true;
+		}
+
+		private void NotifyUser(string message)
+		{
+			statusBlock.Text = message;
+		}
+
+		//private byte ENote = 64;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this commented out code.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5305560)